### PR TITLE
Stop prefixing SVG fill

### PR DESCRIPTION
**Pull request by oliverjam**
Check we are actually prefixing `filter` and not `fill`

Relates #61

### CODE REVIEW

File: stylis.js [hunk 1]
```diff
@@ -958,9 +958,13 @@
 					out = webkit + out.replace(gradientptn, webkit+'$1') + out
 					break
 				}
-				// filter
+				// fill/filter, f, i, l
 				case 951: {
-					out = webkit + out + out
+          // filter
+          if (out.charCodeAt(3) === 116) {
+            console.log(out, out.charCodeAt(5));
```
Comment (thysultan) [inline stylis.js]: oops :)

Comment (oliverjam) [reply stylis.js]: Haha sorry, should be fixed

Comment (oliverjam) [reply stylis.js]: Oh god my editor's Prettier has gone crazy on all the linebreaks, let me fix that quickly

Comment (oliverjam) [reply stylis.js]: Atom _really_ likes messing with whitespace. All sorted now 👍

**Comment by thysultan** [comment]
Thanks Oliver, will push the patch release later today.

**Pull request merged**